### PR TITLE
"AWAvenue-Ads-Rule" request add to "HostlistsRegistry"

### PR DESCRIPTION
**Pull request by TG-Twilight**
I followed the relevant instructions and added the necessary files. I hope that "AWAvenue-Ads-Rule" can be added to the "HostlistsRegistry". Thank you very much！！

### CODE REVIEW

**Review by Sergey-Lyapin — COMMENTED** [review]
Hello @TG-Twilight
You created files in the regional folder, so this filter is targeting Chinese language sites? Or does it target the Internet in general?

File: filters/regional/filter_54_AWAvenue-Adblock-RuleRule/filter.txt [hunk 1]
```diff
@@ -0,0 +1,362 @@
+[-AWAvenue 秋风广告规则（AWAvenue-Adblock-Rule）-]
```
Comment (Sergey-Lyapin) [inline filters/regional/filter_54_AWAvenue-Adblock-RuleRule/filter.txt:1]: Only 2 files should be created in the folder: configuration.json and metadata.json

File: filters/regional/filter_54_AWAvenue-Adblock-RuleRule/configuration.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "秋风广告规则",
```
Comment (Sergey-Lyapin) [inline filters/regional/filter_54_AWAvenue-Adblock-RuleRule/configuration.json:2]: The name and description must be in English here.

File: filters/regional/filter_54_AWAvenue-Adblock-RuleRule/metadata.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "filterId": "AWAvenue-Adblock-Rule",
+    "id": 21,
+    "name": "秋风广告规则",
```
Comment (Sergey-Lyapin) [inline filters/regional/filter_54_AWAvenue-Adblock-RuleRule/metadata.json:4]: The name and description must be in English here.

File: locales/zh/filters.json [hunk 4]
```diff
@@ -159,4 +159,8 @@
 		"hostlist.42.name": "ShadowWhisperer's Malware List",
 		"hostlist.42.description": "恶意网站、PUP、恶意软件、浏览器劫持程序、钓鱼网站"
 	}
```
Comment (Sergey-Lyapin) [inline locales/zh/filters.json:161]: The comma is forgotten here.

And please add also English localization.

Comment (TG-Twilight) [reply locales/zh/filters.json]: It has been modified as required, please review, thank you!

File: filters/regional/filter_54_AWAvenue-Adblock-RuleRule/configuration.json (hunk 2 above)
Comment (TG-Twilight) [reply filters/regional/filter_54_AWAvenue-Adblock-RuleRule/configuration.json]: It has been modified as required, please review, thank you!

File: filters/regional/filter_54_AWAvenue-Adblock-RuleRule/filter.txt (hunk 1 above)
Comment (TG-Twilight) [reply filters/regional/filter_54_AWAvenue-Adblock-RuleRule/filter.txt]: Deleted as requested, please review, thank you！

**Comment by TG-Twilight** [comment]
> 你好@TG-Twilight 您在区域文件夹中创建了文件，因此该过滤器针对的是中文网站？或者它是否针对整个互联网？

Yes, currently we basically only filter advertising content on the Chinese Internet. In the future, I will filter the entire Internet.

**Comment by TG-Twilight** [comment]
> > 你好@TG-Twilight 您在区域文件夹中创建了文件，因此该过滤器针对的是中文网站？或者它是否针对整个互联网？
> 
> Yes, currently we basically only filter advertising content on the Chinese Internet. In the future, I will filter the entire Internet.
(In fact, the main function of this advertising rule is to directly block advertising content inside Chinese apks. Currently, it can perfectly block most advertising inside apk)

In the meantime, I wish you a happy life!😀

**Pull request closed without merging**